CHANGELOG.md

```diff
@@ -2,6 +2,21 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x will End of Life on September 8th, 2021.** Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) to plan your migration to the latest 2.x version. Support for the 1.x version ended on December 4th, 2020.
 
+2.15.3 (July 28, 2021)
+======================
+
+Bug Fixes
+---------
+Chrome 92 [started enforcing](https://chromium-review.googlesource.com/c/chromium/src/+/2816118) limit on number of WebMediaPlayers. This blocks creation of  WebMediaPlayers once there are already 75 (desktop) or 40 (mobile) players. Fixed a related bug where the SDK was not cleaning up internally maintained media elements.
+Please ensure that your application cleans up media elements as well after they are detached (VIDEO-6336).
```

Comment:
We should get another pair of eyes on this changelog. @anna-vasilko @donaltoomey @syerrapragada 

I think we need to reword it such that this change is not really fixing the chrome 92 issue. It is only helping prevent that issue on certain scenarios like reconnecting or republishing. What do you think of this? Feel free to change it.

> Fixed a bug where the SDK was not cleaning up internally maintained media elements. This causes memory leaks on certain use cases such as reconnecting or republishing to a room.
>
> Additionally, Chrome 92 [started enforcing](https://chromium-review.googlesource.com/c/chromium/src/+/2816118) limit on number of WebMediaPlayers. This blocks creation of  WebMediaPlayers once the limit is reached - 75 for desktop and 40 for mobile. This SDK update will help prevent running into this limit issue on use cases such as reconnecting or republishing to a room. Please ensure that your application cleans up media elements as well after they are detached.
>
>```js
>  const elements = track.detach();
>  elements.forEach(el => {
>    el.remove();
>    el.srcObject = null;
>  });
>```
>
> Please be aware that your application may still run into the Chrome's WebMediaPlayers limit for large rooms where participants exceeds this limit.

I think we need to add information on resolutions we're investigating for large rooms.

Reply:
Thanks @charliesantos, I like your text. I will update.